roadmap.md

```diff
@@ -176,7 +176,12 @@ trying to make it faster, more predictable, and to remove some confusions and in
 
 There is almost no use case in which Apache Flink is used on its own. It has established itself
 as part of many data related reference architectures. In fact you'll find the squirrel logo covering
-several aspects. The community has added a lot of connectors and formats. With the already mentionend
+several aspects. 
+
+The Flink community has removed Gelly, it's old graph-processing library. There are plans for replacing Gelly based on 
+the new iterations framework that is available via [Flink ML](https://nightlies.apache.org/flink/flink-ml-docs-stable/docs/development/iteration/).
```

Comment:
If we write down that there _are plans_ then we better have a place to collect ideas etc. and some people who to signed up for it.

Reply:
Note that IMO this is also way too technical for users.

Reply:
> If we write down that there _are plans_ then we better have a place to collect ideas etc. and some people who to signed up for it and a timeline.

I agree. I've pointed to the people on the Dev ML that they should reply on this PR and they should provide input:
- Where ideas can be shared
- Who can be contacted
- What is an expected timeline

If such info can't be provided, that I don't think it makes much sense to include a note on continuation of graph-processing report.